cypress/integration/e2e/table/range-selection.spec.ts

```diff
@@ -288,7 +239,7 @@ context("Shift range selection", () => {
           .eq(dataKey)
           .children()
           .eq(dayIdx)
-          .contains(desiredNumber);
+          .should("contain", desiredNumber);
```

Comment:
Tutaj doznałem odkrycia, że część naszych testów jest błędnych. Zamiast używać asserci np. should, która sprawdza warunek my używamy contains, które nic nie sprawdza, a używane jest tylko do wyszukiwania elementów na stronie